### ISSUE TITLE
AWSQS::Kubernetes::Resource can't recreate the kubernetes-resource-proxy after deleting an addon

### ISSUE THREAD

**Issue by tomiszili** [aws-quickstart/quickstart-amazon-eks#452]
Hello!

The AWSQS::Kubernetes::Resource deletes the kubernetes-resource-proxy after deleting a manifest with them.
In my case i deployed the metrics server addons and delete the addon for test purposes, but when I want to deploy it again the CloudFormation fails. I paste the logs of the `awsqs-kubernetes-resource` and it can't find the `kubernetes-resource-proxy-<clustername>` lambda because it was deleted from the previous CloudFormation addon delete. And the next time it fails to recreate the function.

I think some modification makes the zip files too large for the lambda_create function `ZipFile` parameter.

```
Exception caught An error occurred (RequestEntityTooLargeException) when calling the CreateFunction operation: Request must be smaller than 69905067 bytes for the CreateFunction operation
Traceback (most recent call last):
  File "/var/task/awsqs_kubernetes_resource/vpc.py", line 116, in put_function
    no_update = update_function_config(lmbd, function_config)
  File "/var/task/awsqs_kubernetes_resource/vpc.py", line 131, in update_function_config
    lmbd.update_function_configuration(**function_config)
  File "/var/task/botocore/client.py", line 514, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/var/task/botocore/client.py", line 938, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.errorfactory.ResourceNotFoundException: An error occurred (ResourceNotFoundException) when calling the UpdateFunctionConfiguration operation: Function not found: arn:aws:lambda:eu-central-1:201397412937:function:awsqs-kubernetes-resource-proxy-Demo-1v21

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/var/task/cloudformation_cli_python_lib/resource.py", line 221, in __call__
    raise error
  File "/var/task/cloudformation_cli_python_lib/resource.py", line 214, in __call__
    progress = self._invoke_handler(caller_sess, request, action, callback)
  File "/var/task/cloudformation_cli_python_lib/resource.py", line 94, in _invoke_handler
    progress = handler(session, request, callback_context)
  File "/var/task/awsqs_kubernetes_resource/handlers.py", line 74, in create_handler
    session,
  File "/var/task/awsqs_kubernetes_resource/utils.py", line 53, in run_command
    put_function(session, cluster_name)
  File "/var/task/awsqs_kubernetes_resource/vpc.py", line 121, in put_function
    lmbd.create_function(Code={"ZipFile": zip_file.read()}, **function_config)
  File "/var/task/botocore/client.py", line 514, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/var/task/botocore/client.py", line 938, in _make_api_call
    raise error_class(parsed_response, operation_name)
botocore.exceptions.ClientError: An error occurred (RequestEntityTooLargeException) when calling the CreateFunction operation: Request must be smaller than 69905067 bytes for the CreateFunction operation
```

**Comment by gargana** [comment]
I have reproduced and am working on a fix for this.

**Comment by tomiszili** [comment]
Okay the above merge solved the recreation of the kubernetes-resource-proxy. But when I deplyo the whole EKS quickstart templates the FunctionStack creates an initial version of this k8 resource proxy function. And I got several errors when this inital lambda whant to create a manifest (e.g.: metrics-server). The errors can be found in [this issue comment](https://github.com/aws-quickstart/quickstart-kubernetes-resource-provider/issues/33).
After I deleted the function manually from the Lambda console and created the CloudFormation stack again it deployed the manifest properly without any error.

Any suggestion how to solve thiese errors of the initial function deployed by the FunctionStack?
